node/mock-answer.ts

```diff
@@ -22,18 +22,67 @@ export interface TaskLibAnswers {
     rmRF?: { [key: string]: { success: boolean } },
     stats?: { [key: string]: any }, // Can't use `fs.Stats` as most existing uses don't mock all required properties
     which?: { [key: string]: string },
+    variables?: {
```

Comment:
It seemed odd to require, or even allow, people to mock `getVariable` and `getVariables` separately. So I changed the pattern slightly and take in a collection of variables that will be used by both. `getVariables` needs to know which variables are secret.

Reply:
Makes sense to me, I like this pattern more.